packages/react-ui-core/src/DropDownMenu/__tests__/DropDownMenu-test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import DropDownMenu from '../DropDownMenu'
+
+const theme = {
+  DropDownMenu: 'DropDownMenu',
+  DropDownMenu_ListItem: 'DropDownMenu_ListItem',
+  DropDownMenu_Selected: 'DropDownMenu_Selected',
+}
+
+const items = [1, 2, 3]
+
+describe('DropDownMenu', () => {
+  const setup = props => {
+    const wrapper = mount(
```

Comment:
Do we need to `mount` here or can we use `shallow`?

Reply:
Just doubled checked, looks like it needs to be a full `mount`